modules/core/server/controllers/core.dbmodel.controller.js

```diff
@@ -1021,7 +1025,8 @@ _.extend (DBModel.prototype, {
 	},
 
 	paginate: function(query, filter, skip, limit, fields, population, sortby, userCan) {
-		// console.log ('paginate(query=', query, ', filter=', filter, ', skip=', skip, ', limit=', limit, ', fields=', fields, ', population=', population, ', sortby=', sortby, ', userCan=', userCan, ')');
+		//console.log ('paginate(query=', query, ', filter=', filter, ', skip=', skip, ', limit=', limit, ', fields=', fields, ', population=', population, ', sortby=', sortby, ', userCan=', userCan, ')');
+		var debug = false;
```

Comment:
This is meant to control some developer optional debug output.  I'm not aware of the build system and controlling output via env vars. So ... how about I just comment out or delete the debug output?